# Causal_GE
This is the official repository associated to the paper **Reanalyzing gene-expression data from a causal perspective**

Differential expression and co-expression analysis [1, 2], pathway analysis [3], and dimensional reduction techniques [4] are usual ways of processing gene expression data.
As a result, we obtain sets of differentially expressed genes and enriched pathways, which are assumed to underly the main biological processes in the system under study.
In the present paper, we revisit gene-expression data by looking at it from a causal perspective.  

[1]-Anders, S., Huber, W. Differential expression analysis for sequence count data. Nat. Prec. (2010). https://doi.org/10.1038/npre.2010.4282.1  
[2]-Sipko van Dam, Urmo Võsa, Adriaan van der Graaf, Lude Franke, João Pedro de Magalhães. Gene co-expression analysis for functional classifi7
cation and gene–disease predictions. Briefings in Bioinformatics, Volume 19, Issue 4, 2018, Pages 575–592. https://doi.org/10.1093/bib/bbw139  
[3]-Emmert-Streib F, Glazko GV (2011). Pathway Analysis of Expression Data: Deciphering Functional Building Blocks
of Complex Diseases. PLoS Comput Biol 7(5): e1002053. https://doi.org/10.1371/journal.pcbi.1002053

# Repository sections
### Data
This folder includes all data files that are not generated as a result of our analyses, including the graph file and files to map between nodes and
gene names/ESEMBL ids

### result_files
Folder to allocate all files resulting from our analysis, excluding charts.

### Charts
Contains every generated figure 
